### ISSUE TITLE
forced to redraw ui component when used with shinyjs

### ISSUE THREAD

**Issue by happyshows** [rstudio/shiny#1528]
Hi,

I'm seeing a strange behavior when using nested module approach with shinyjs. Please run the minimal example listed in [here](https://github.com/daattali/shinyjs/issues/111)

**Comment by daattali** [comment]
I'd like to mention that from what I understand your code is trying to do, this isn't a bug in shinyjs or in shiny. I think what you're seeing is the entire element inside `renderUI` getting redrawn, which is the correct behaviour, although perhaps not your desired behaviour.

**Comment by happyshows** [comment]
@daattali I do not agree on your statement that updating the attribute of element & cause renderUI to redraw is the correct behavior. There's no reactive dependency defined that should cause this.

**Comment by daattali** [comment]
You might be right, I didn't dive deep into your code, it's what I thought from a quick look.  Someone from the shiny team can look at your code and see what they think
